### ISSUE TITLE
refmt removes [ and ] for `[%bs.raw "..."]`

### ISSUE THREAD

**Issue by arnarthor** [reasonml/reason#1848]
I have this code
```ocaml
beforeEach(() =>
  [%bs.raw
    {|
        global.console = {log: console.log, warn: jest.fn()}
    |}
  ]
);
```

Which refmt's to this with the latest update
```ocaml
beforeEach(() =>
  %bs.raw
  {|
        global.console = {log: console.log, warn: jest.fn()}
    |}
);
```

Which results in an invalid syntax for `%bs.raw`

**Comment by chenglou** [comment]
cc @let-def could this be caused by the infix extension point sugar change?

**Comment by let-def** [comment]
@chenglou Likely yes.

Should the parser accepts:
```reason
beforeEach(() =>
  %bs.raw
  {|
        global.console = {log: console.log, warn: jest.fn()}
    |}
);
```
Or should the printer keep `[` `]` in this case?

**Comment by chenglou** [comment]
I don't believe there's a precent for brackets-less extension points right? Except for the extension point sugar like `let%foo`.
If that's the case, then we should keep the brackets

**Comment by let-def** [comment]
I am not sure what you mean? Brackets-less extension points always start with a percent.

**Comment by let-def** [comment]
For instance:
```reason
Reason # %a "lol";
Error: Uninterpreted extension 'a'.
```
This is valid syntax (there is a semantic error of course after, but that's not the point).

**Comment by chenglou** [comment]
Oh boy, I've been using the brackets all this time, didn't even realize we had brackets-less extension points syntax...

I'm on the fence. What do you think?

**Comment by davesnx** [comment]
Pretty old issue, but will be fixed soon after this https://github.com/reasonml/reason/pull/2674 gets merged.

Probably 3.8.3

**Comment by anmonteiro** [comment]
fixed by #2674
